### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by es183923**
This PR fixes some typos, including an enum, and a variable.

### CODE REVIEW

**Comment by braindigitalis** [comment]
this pr is scary massive, not sure how to merge it without it conflicting with other changes in progress rn...

**Comment by es183923** [comment]
I've removed the change for removing trailing fixes.

File: include/dpp/user.h [hunk 1]
```diff
@@ -56,7 +56,7 @@ enum user_flags : uint32_t {
 	/// User is a member of House Brilliance
 	u_house_brilliance =		0b00000000000100000000000,
 	/// User is a member of House Balance
-	u_house_balanace =		0b00000000001000000000000,
+	u_house_ =		0b00000000001000000000000,
```
Comment (braindigitalis) [inline include/dpp/user.h:59]: u_house what exactly 😂

File: include/dpp/user.h [hunk 2]
```diff
@@ -204,7 +204,7 @@ class DPP_EXPORT user : public managed {
 	 *
 	 * @return true if in house brilliance
 	 */
-	bool is_house_balanace() const;
+	bool is_house_() const;
```
Comment (braindigitalis) [inline include/dpp/user.h:207]: hooooooose

**Review by braindigitalis — CHANGES_REQUESTED** [review]

**Comment by braindigitalis** [comment]
any progress on this? just needs u_house_ fixing and it can be merged in

**Pull request merged**